### DISCUSSION TITLE
Verifying container external IP

### DISCUSSION THREAD

**Discussion by recruiterguy** [qdm12/gluetun#1339] [Q&A] [answered]
I realize this might be a rookie question, but I've just started trying to use Gluetun and Portainer on a Synology NAS and have found that in Portainer the console access for /bin/bash won't connect.  I was hoping that someone could help me by telling me another way to validate if the Gluetun container is working properly with PIA.

**Comment by qdm12** [comment, marked as answer]
The shell in Gluetun (based on alpine) is /bin/sh

If gluetun reports it's healthy, it means it's connected and everything is secured (unless you disable the firewall which is undocumented because you shouldn't 😉). It should also log out your public IP information which is obtained from ipinfo.io

**Reply by recruiterguy** [reply]
Thanks!  I certainly didn't do anything "extra" with the installation and configuration so I would think the fw is intact, lol.
I did just find that if I go into the docker log for Gluetun I can see the second to last line each time it loads tells me what IP address it's working on.
My next step is to figure out how to get the other container to work through that! :-)
